app/poros/food.rb

```diff
@@ -0,0 +1,6 @@
+class Food
+  attr_reader :description
+  def initialize(data)
+    @description = description
```

Comment:
I think this would be 
```
@description = data[:description]
```
We would also want upc (GTIN/UPC code), brandOwner, ingredients in here